automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -569,67 +569,26 @@ trait LeonardoTestUtils
   def stopAndMonitorRuntime(googleProject: GoogleProject, runtimeName: RuntimeName)(implicit token: AuthToken): Unit =
     stopRuntime(googleProject, runtimeName, monitor = true)(token)
 
-  def startCluster(googleProject: GoogleProject, clusterName: RuntimeName, monitor: Boolean)(
-    implicit token: AuthToken
+  def startAndMonitorRuntime(googleProject: GoogleProject, runtimeName: RuntimeName)(
+    implicit token: AuthToken,
+    authorization: Authorization
   ): Unit = {
-    Leonardo.cluster.start(googleProject, clusterName)(token) shouldBe
-      "The request has been accepted for processing, but the processing has not been completed."
-
     // verify with get()
-    val startingCluster = Leonardo.cluster.get(googleProject, clusterName)
-    startingCluster.status shouldBe ClusterStatus.Starting
-
-    if (monitor) {
-      // wait until in Running state
-      implicit val patienceConfig: PatienceConfig = clusterPatience
-      eventually {
-        val status = Leonardo.cluster.get(googleProject, clusterName).status
-        status shouldBe ClusterStatus.Running
-      }
-
-      logger.info(s"Checking if cluster is proxyable yet")
-      val getResult = Try(Notebook.getApi(googleProject, clusterName))
-      getResult.isSuccess shouldBe true
-      getResult.get should not include "ProxyException"
-
-      // Grab the jupyter.log and welder.log files for debugging.
-      saveClusterLogFiles(googleProject, clusterName, List("jupyter.log", "welder.log"), "start")
+    val waitForRunning = LeonardoApiClient.client.use { implicit c =>
+      LeonardoApiClient.startRuntimeWithWait(googleProject, runtimeName)
     }
-  }
 
-  def startRuntime(googleProject: GoogleProject, runtimeName: RuntimeName, monitor: Boolean)(
-    implicit token: AuthToken
-  ): Unit = {
-    Leonardo.cluster.startRuntime(googleProject, runtimeName)(token) shouldBe
-      "The request has been accepted for processing, but the processing has not been completed."
+    waitForRunning.unsafeRunSync()
 
-    // verify with get()
-    val startingCluster = Leonardo.cluster.getRuntime(googleProject, runtimeName)
-    startingCluster.status shouldBe ClusterStatus.Starting
-    if (monitor) {
-      // wait until in Running state
-      implicit val patienceConfig: PatienceConfig = clusterPatience
-      eventually {
-        val status = Leonardo.cluster.getRuntime(googleProject, runtimeName).status
-        status shouldBe ClusterStatus.Running
-      }
-
-      logger.info(s"Checking if runtime is proxyable yet")
-      val getResult = Try(Notebook.getApi(googleProject, runtimeName))
-      getResult.isSuccess shouldBe true
-      getResult.get should not include "ProxyException"
+    logger.info(s"Checking if runtime is proxyable yet")
+    val getResult = Try(Notebook.getApi(googleProject, runtimeName))
+    getResult.isSuccess shouldBe true
+    getResult.get should not include "ProxyException"
```

Comment:
If the assertions above fail, will we know what the runtime name is? I think we were saying that'd be useful to know during the walkthrough today.

Reply:
probably not 🤔  this is copied straight from existing code..I can add runtimeName on line 583